### PR TITLE
Added used models to Feature-extraction #355

### PR DESCRIPTION
**Pull request by anmar01**
New folder for feature-extraction added, and added the used models for this taks into data.ts

### CODE REVIEW

**Comment by merveenoyan** [comment]
Partially fixes #355

**Comment by merveenoyan** [comment]
hello @anmar01 , do you need any help?

**Comment by anmar01** [comment]
Hi @merveenoyan, I was waiting for your review but I see now that you have already submitted it

**Comment by merveenoyan** [comment]
@anmar01 can you ask for review when you address my comments? It would be good 🙂

**Comment by anmar01** [comment]
@merveenoyan I don't see any comments from you, and on this page it's still pending review. Maybe I'm doing something wrong?
image

File: tasks/src/feature-extraction/about.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Use Cases
```
[the remark this answers is not in the document]
Comment (anmar01) [reply tasks/src/feature-extraction/about.md]: removed

**Review by anmar01 — COMMENTED** [review]
@merveenoyan comments addressed in the PR

**Comment by merveenoyan** [comment]
Is it okay if you could fix merge conflict? @anmar01 then I'll review!

**Pull request closed without merging**